### PR TITLE
[TEST] - Json output step was needed for tests but was no longer in k…

### PR DESCRIPTION
**Pull request by pamval**
…ettle-core. Adding explicit dependency and loading the plugin in test init

### CODE REVIEW

**Comment by pamval** [comment]
@graimundo @pedrofvteixeira please review

**Comment by pamval** [comment]
@graimundo @pedrofvteixeira please review

File: cpk-core/ivy.xml [hunk 1]
```diff
@@ -48,9 +48,14 @@
     <dependency org="com.sun.jersey" name="jersey-client" rev="1.16" conf="test->default" transitive="false"/>
 
 
+
     <!--  third-party dependencies for tests -->
     <dependency org="com.google.guava" name="guava" rev="17.0" conf="test->default"/>
-     <dependency org="pentaho"             name="metastore"           rev="${dependency.pentaho-metastore.revision}"  conf="test->default" changing="true" />
+    <dependency org="pentaho"             name="metastore"           rev="${dependency.pentaho-metastore.revision}"  conf="test->default" changing="true" />
+    <dependency org="pentaho-kettle" name="kettle-json-plugin" rev="${dependency.kettle.revision}" transitive="false" conf="test->default"  changing="true">
```
Comment (graimundo) [inline cpk-core/ivy.xml]: There is a concrete version property for the kettle-json-plugin => kettle-json-plugin.version. Please update the version property name.
http://devci.pentaho.com/job/7.1-snapshot/view/control/job/build-resources/ws/7.1-SNAPSHOT/version.properties/*view*/

File: cpk-core/ivy.xml [hunk 2]
```diff
@@ -48,9 +48,14 @@
...
+      <artifact name="kettle-json-plugin" type="jar" ext="jar" conf=""/>
```
Comment (graimundo) [inline cpk-core/ivy.xml]: Just a curiosity... I have no idea why but the kettle-json-plugin jar artifact is not showing when searching for it in nexus: http://nexus.pentaho.org/#nexus-search;gav~pentaho-kettle~kettle-json-plugin~~~~kw,versionexpand
However it does resolve properly and it appears in the maven metadata http://nexus.pentaho.org/content/groups/omni/pentaho-kettle/kettle-json-plugin/7.1-SNAPSHOT/maven-metadata.xml. Don't know if its worth it to raise the question to the build team.

Comment (graimundo) [inline cpk-core/ivy.xml]: PS: I know this is IVY but the artifacts metadata should be synchronized (or maybe it's even a passthrough?). I don't know how to directly check Ivy metadata info and http://ivy-nexus.pentaho.org/ redirects to http://nexus.pentaho.org/

**Comment by pamval** [comment]
@graimundo updated

**Pull request merged**